kahuna/public/js/errors/global.html

```diff
@@ -16,7 +16,7 @@
 
         <div>
             We are looking into this but if the problem persists please
-            <a href="mailto:thegrid.dev@theguardian.com"
+            <a href="{{ctrl.supportEmailLink}}"
```

Comment:
Would something like this make config simpler? That is, remove `mailto` from the config. Should also mean this property can be used elsewhere if necessary.

(I'm guessing at the syntax here, as don't have an IDE to help 😄 )

```suggestion
            <a href="{{`mailto:${ctrl.supportEmail}`}}"
```

Reply:
oh, we do this in a few places... feel free to leave as is and we can change later if it becomes an issue.